<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleTable</class>
 <widget class="QMainWindow" name="VehicleTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vehicle Table</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MYPICTURES.qrc">
    <normaloff>:/picture/Chevorlet.png</normaloff>:/picture/Chevorlet.png</iconset>
  </property>
  <property name="toolTip">
   <string>Vehicle Table</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 170, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="8">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>32</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Vehicle table</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="8">
       <widget class="QTableView" name="tableView_vehicle">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>157</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0" colspan="8">
       <widget class="QLabel" name="label_id">
        <property name="font">
         <font>
          <pointsize>32</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Update vehicle data</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="7" rowspan="10">
       <widget class="QLabel" name="label_picture">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Manufacturer</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_manufacturer"/>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Construction year</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_construction_year"/>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Km stood</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_km_stood"/>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Condition</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_condition"/>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Pieces</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_pieces"/>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_price"/>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Currency</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_currency"/>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Availability</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2" colspan="5">
       <widget class="QLineEdit" name="lineEdit_availability"/>
      </item>
      <item row="13" column="0" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="3">
       <widget class="QPushButton" name="pushButton_update_picture">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 255, 0);</string>
        </property>
        <property name="text">
         <string>Uptade vehicle picture</string>
        </property>
       </widget>
      </item>
      <item row="13" column="4">
       <widget class="QPushButton" name="pushButton_update_data">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Update vehicle data</string>
        </property>
       </widget>
      </item>
      <item row="13" column="5">
       <widget class="QPushButton" name="pushButton_delete_data">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>Delete vehicle data</string>
        </property>
       </widget>
      </item>
      <item row="13" column="6">
       <widget class="QPushButton" name="pushButton_homePage">
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 85, 255);</string>
        </property>
        <property name="text">
         <string>Home Page</string>
        </property>
       </widget>
      </item>
      <item row="13" column="7" rowspan="2">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="3">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="4">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="5">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="6">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="2">
       <widget class="QPushButton" name="pushButton_upload_picture">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Upload vehicle picture</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MYPICTURES.qrc"/>
 </resources>
 <connections/>
</ui>
